accounts/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class Account(models.Model):
+    email = models.EmailField(max_length=254, blank=True)
+    first_name = models.CharField(max_length=50, blank=False)
+    last_name = models.CharField(max_length=50, blank=True)
+    mobile_number = models.CharField(max_length=12, blank=True)
```

Comment:
Either use `PhoneNumberField` or validate via `regex`

Reply:
This model is needed for getting the API of Google OAuth token, some people may have saved mobile number not starting with +91 so PhoneNumberField won't accept those phone numbers.

Reply:
Yeah that's why use regex, make `+91` optional

Reply:
okay, I will work on it.